_pytest/main.py

```diff
@@ -175,7 +175,7 @@ def pytest_ignore_collect(path, config):
     if excludeopt:
         ignore_paths.extend([py.path.local(x) for x in excludeopt])
 
-    if path in ignore_paths:
+    if py.path.local(path) in ignore_paths:
```

Comment:
pytest-dev/py#128 has been merged, should we merge this one as well @RonnyPfannschmidt?